speed_to_erpm_gain: 64.9612012619981
speed_to_erpm_offset: 0
steering_angle_to_servo0_gain: 0.5
steering_angle_to_servo0_offset: 0.4
steering_angle_to_servo1_gain: 0.5
steering_angle_to_servo1_offset: 0.5
wheelbase: 0.325